app/models/katello/erratum.rb

```diff
@@ -25,7 +25,7 @@ class Erratum < Katello::Model
     has_many :packages, :class_name => "Katello::ErratumPackage", :dependent => :destroy, :inverse_of => :erratum
 
     scoped_search :on => :errata_id, :only_explicit => true
-    scoped_search :on => :errata_id, :rename => :id, :complete_value => true, :only_explicit => true
+    scoped_search :on => :id, :rename => :db_id, :complete_value => true, :only_explicit => true, :validator => ScopedSearch::Validators::INTEGER
```

Comment:
`:complete_value => true` means that autocomplete will suggest values for it when searching in the UI. I don't find this useful for database ID, but I think we should have it for `errata_id`.